src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -743,6 +743,61 @@ int shim::p2pEnable(bool enable, bool force)
     return p2p_enable;
 }
 
+int shim::cmaEnable(bool enable, uint64_t size, uint64_t num, bool force)
+{
+    const std::string input = "1\n";
+    std::string err;
+    int ret = 0;
+    uint64_t i = 0;
+    void *addr_local = NULL;
+    uint32_t hugepage_flag = 0;
+
+    if (enable) {
+
+        drm_xocl_alloc_cma_info cma_info;
+        cma_info.page_sz = size;
+        cma_info.nr_page = num;
+        cma_info.reserve = true;
+
+        if (num > 4 || num == 0)
+            return -EINVAL;
+
+        if (size == (1 << 30)) 
+            hugepage_flag = 0x1e;
```

Comment:
what do do these constant values mean?

Reply:
> 
> 
> that's huge page size in log, 0x1e = 30, 2^30 = 0x40000000 = 1G,
> 
> Once set MAP_HUGETLB, we have to specify bit[26~31] as size

We either need a block of comment to explain 0x1e or a well-named const variable to replace 0x1e here. It's difficult to understand what is happening here with just a const value - 0x1e.